### DISCUSSION TITLE
cac:PostalAddress & cac:Contact

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#163] [Q&A] [answered]
Hi !
in xsd CompanyType ( as in Touchpoint) there is one ( not mandatory ) PostalAddress and Contact
<xsd:complexType name="CompanyType">
		<xsd:sequence>
*************************************************************************************
			<xsd:element ref="cac:PostalAddress" minOccurs="0" maxOccurs="1"/>
**************************************************************
			<xsd:element ref="cac:Contact" minOccurs="0" maxOccurs="1"/>
		</xsd:sequence>

in Conceptual Model , under ND-Company and ND-Touchpoint there are many fields like 
BT-513-Organization-Company											cac:PostalAddress/cbc:CityName
BT-512-Organization-Company											cac:PostalAddress/cbc:PostalZone
BT-507-Organization-Company											cac:PostalAddress/cbc:CountrySubentityCode
BT-514-Organization-Company											cac:PostalAddress/cac:Country/cbc:IdentificationCode

BT-502-Organization-Company											cac:Contact/cbc:Name
BT-506-Organization-Company											cac:Contact/cbc:ElectronicMail
BT-503-Organization-Company											cac:Contact/cbc:Telephone
BT-739-Organization-Company											cac:Contact/cbc:Telefax

What do you think to set a node for PostalAddress and the Contact ? 
Just to avoid some ugly "if" in the code ..
Thanks
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi Renato
This has been done and should be part of the next release
